### ISSUE TITLE
Can not convert pdbqt to mol2

### ISSUE THREAD

**Issue by He-Ziping** [openbabel/openbabel#2758]
## Environment Information
Open Babel version:
Open Babel 3.1.1
Operating system and version:
Windows 11

## Expected Behavior

convert pdbqt to mol2
## Actual Behavior
my command line: 
PS C:\Document> obabel -ipdbqt docking_results.pdbqt -O dockout.mol2 -m
result:
obabel: cannot read input format!
Open Babel 3.1.1 -- May 16 2020 -- 11:57:55

second times:PS C:\Document> obabel docking_result.pdbqt -O dockout.mol2 -m
*** Open Babel Error  in OpenBabel::OBConversion::OpenAndSetFormat
  Cannot read input format ".pdbqt" for file "docking_result.pdbqt"
0 molecules converted

**Comment by nbehrnd** [comment]
Does the conversion only fail with the specific file `docking_results.pdbqt`?  Do you have clearance to share the pdbqt file (put it into a zip archive, use the paperclip icon here on GitHub's interface to upload it) to replicate your attempt?

Sidenote: By file extension, OpenBabel infers the file format, i.e. the syntax of the file's content to work with.  Hence in an instance like here, `-ipdbqt` (where `-i` is about input, and `pdbqt` specifies the format, see [the documentation](https://open-babel.readthedocs.io/en/latest/Command-line_tools/babel.html#synopsis)) is not required.  By similar token, your usage of `-O dockout.mol2` describes both the file name of the new record to be written, and file format -- without the need to use an extra `-omol2`, too.

**Comment by fredrikw** [comment]
There seems to be a problem with the installation since you can't seem to read pdbqt files. If you do obabel -L formats what does it print? Specifically, is pdbqt in the list? There might be a problem with the BABEL_LIBDIR setting.

**Comment by He-Ziping** [comment]
> There seems to be a problem with the installation since you can't seem to read pdbqt files. If you do obabel -L formats what does it print? Specifically, is pdbqt in the list? There might be a problem with the BABEL_LIBDIR setting.

You’re right! I couldn’t find the PDBQT format in the list. Could you please share some advice on how to resolve the BABEL_LIBDIR setting?

**Comment by He-Ziping** [comment]
> I just fetched the OpenBabel-3.1.1-x64.exe installer from [the releases page](https://github.com/openbabel/openbabel/releases) for an installation in Windows 10. Running `obabel -L formats` includes the entry about pdbqt. Does a deinstallation of the maybe broken/incomplete installation and subsequent fresh re-installation (md5sum 60a05f3ab8ea2ff2041c9ae488246b09) solve the issue for you?
> 
> For a reason unknown to me (separate issue #2759), `obabel --version` reports `Open Babel 3.1.1 -- Oct 15 2024 -- 18:31:13` though.

 This problem not be solved when i reinstalled the openbabel

**Comment by fredrikw** [comment]
How did you install OpenBabel?

**Comment by He-Ziping** [comment]
> How did you install OpenBabel?

I just downloaded the OpenBabel-3.1.1-x64.exe and double click to run and followed the guidance without change any location path.

**Comment by fredrikw** [comment]
And you run obabel in a regular cmd-prompt? If you run `where.exe obabel` what does that give? Normally, the installer should make sure that BABEL_LIBDIR is correct...
